kolena/_api/v2/quality_standard.py

```diff
@@ -17,3 +17,4 @@
 class Path(str, Enum):
     QUALITY_STANDARD = "quality-standard"
     RESULT = "quality-standard/result"
+    COPY_QUALITY_STANDARD_FROM_DATASET = "quality-standard/copy-from-dataset"
```

Comment:
nit: I don't think enum's name needs to include `QUALITY_STANDARD` since the enum is being imported from the quality standard module. Could shorten to `COPY_FROM_DATASET`